javatests/arcs/android/devtools/StoreOperationMessageTest.kt

```diff
@@ -0,0 +1,66 @@
+package arcs.android.devtools
+
+import arcs.android.devtools.DevToolsMessage.Companion.ACTOR
+import arcs.android.devtools.DevToolsMessage.Companion.KIND
+import arcs.android.devtools.DevToolsMessage.Companion.MESSAGE
+import arcs.android.devtools.DevToolsMessage.Companion.OPERATIONS
+import arcs.android.devtools.DevToolsMessage.Companion.STORAGE_KEY
+import arcs.android.devtools.DevToolsMessage.Companion.STORE_ID
+import arcs.android.devtools.DevToolsMessage.Companion.STORE_OP_MESSAGE
+import arcs.android.devtools.DevToolsMessage.Companion.STORE_TYPE
+import arcs.android.devtools.DevToolsMessage.Companion.TYPE
+import arcs.android.devtools.DevToolsMessage.Companion.UPDATE_TYPE
+import arcs.android.devtools.DevToolsMessage.Companion.VALUE
+import arcs.android.devtools.DevToolsMessage.Companion.VERSION_MAP
+import arcs.core.crdt.CrdtData
+import arcs.core.crdt.CrdtOperation
+import arcs.core.crdt.CrdtSingleton
+import arcs.core.crdt.VersionMap
+import arcs.core.data.util.ReferencablePrimitive
+import arcs.core.storage.ProxyMessage
+import arcs.core.util.JsonValue
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+class StoreOperationMessageTest {
+
+  @Test
+  fun toJson() = runBlockingTest {
```

Comment:
Not sure `runBlockingTest` is needed here? Are any of the operations in this test suspending?

Reply:
Removed.